<p>IF an <code>EVALUATE</code> statement has only one <code>WHEN</code> and possibly one <code>WHEN OTHER</code> then it should probably be changed to an <code>IF</code> statement.</p>
<h2>Noncompliant Code Example</h2>
<pre>
TO DO 
</pre>
<h2>Compliant Solution</h2>
<pre>
TO DO 
</pre>
